plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2626,6 +2661,14 @@ private static void validateBucketColumns(ConnectorTableMetadata tableMetadata)
         if (bucketProperty.isEmpty()) {
             return;
         }
+
+        List<String> sortByColumns = HiveTableProperties.getSortBy(tableMetadata.getProperties()).stream()
```

Comment:
static imprt `getSortBy`

Reply:
we have a `getSortBy(Table table)` in this file as well